packages/google_maps_flutter/google_maps_flutter/example/integration_test/src/tiles_inspector.dart

```diff
@@ -22,7 +21,7 @@ void main() {
 }
 
 void runTests() {
-  const double floatTolerance = 1e-8;
+  const double floatTolerance = 1e-6;
```

Comment:
Added bigger tolerance as on Android and iOS tests failed as diff was bigger than 1e-8